WORKSPACE.bazel

```diff
@@ -40,15 +40,16 @@ COMMON_CXX_WARNINGS = [
 bootlin_toolchain(
     name = "gcc_12_2_toolchain",
     architecture = "x86-64",
-    buildroot_version = "2022.08-1_bleeding",
-    extra_cxxflags = [
+    buildroot_version = "bleeding-edge-2022.08-1",
+    extra_cxx_flags = [
         "-std=c++23",
         "-Wduplicated-cond",
         "-Wduplicated-branches",
         "-Wlogical-op",
         "-Wuseless-cast",
         "-Wshadow=compatible-local",  # Equivalent to -Wshadow on llvm. GCC default is stupid.
     ] + COMMON_CXX_FLAGS + COMMON_CXX_WARNINGS,
+    libc_impl = "glibc",
```

Comment:
surprised this isn't the default

Reply:
Want to submit a PR to make it the default?

Reply:
I don't have enough context to know if that's right. We can talk about it next time we meet up.

Reply:
Note for future conversation.

The attribute is introduced here:
https://github.com/oliverlee/bazel_bootlin/blob/main/defs.bzl#L110

and it is used to specify the toolchain to download (glibc, musl, uclibc).